### PR TITLE
PWGEM: New cut setting and adaptation of old ones

### PR DESCRIPTION
**Pull request by feisenhu**
Adapt cut setting to select e+/e- tracks separately and tracks in +/- eta range.

### CODE REVIEW

**Review by dsekihat — APPROVED** [review]

File: PWGDQ/Core/CutsLibrary.cxx [hunk 1]
```diff
@@ -2022,50 +2040,50 @@ AnalysisCut* o2::aod::dqcuts::GetAnalysisCut(const char* cutName)
...
+    cut->AddCut(VarManager::kTPCnSigmaPr, -3., 3., true, VarManager::kPin, 0.0, 2e+10, false);
     return cut;
   }
 
   if (!nameStr.compare("electronPID_TPCnsigma_corr")) {
-    cut->AddCut(VarManager::kTPCnSigmaEl_Corr, -3., 2., false, VarManager::kPin, 0.0, 1e+10, false);
-    cut->AddCut(VarManager::kTPCnSigmaPi_Corr, -3., 3.5, true, VarManager::kPin, 0.0, 1e+10, false);
-    cut->AddCut(VarManager::kTPCnSigmaKa, -3., 4., true, VarManager::kPin, 0.0, 1e+10, false);
-    cut->AddCut(VarManager::kTPCnSigmaPr_Corr, -3., 4., true, VarManager::kPin, 0.0, 1e+10, false);
+    cut->AddCut(VarManager::kTPCnSigmaEl_Corr, -3., 2., false, VarManager::kPin, 0.0, 2e+10, false);
```
Comment (dsekihat) [inline PWGDQ/Core/CutsLibrary.cxx:2051]: Hi Florian, just a quick question. why do you have to change 1e+10 to 2e+10? Both are extremely large number, right?

Comment (feisenhu) [reply PWGDQ/Core/CutsLibrary.cxx:2051]: Hey Daiki, when we ran over the full statistic of the LHC22o data set we increased the range of a couple of histograms (mee, pt, etc.) But I noticed when looking at some QA plots that the single track pt was cut at 10 GeV/c. To make sure we use the full range of the histograms we would need to increase the acceptance in pt and p.

Comment (dsekihat) [reply PWGDQ/Core/CutsLibrary.cxx:2051]: 1e+10 does not corresponds to 10 GeV/c. it is 1000000000 GeV/c.

Comment (feisenhu) [reply PWGDQ/Core/CutsLibrary.cxx]: Yes you are absolutely right. This was my mistake. Sorry.

**Review by dsekihat — APPROVED** [review]

**Pull request merged**